deploy/addon/postgresql/pgpool.md

```diff
@@ -0,0 +1,347 @@
+
+## Tips / debug
+
+#### How can I access Pgpool-II from my application?
+Connect via [ssh](https://www.clever-cloud.com/doc/administrate/ssh-clever-tools/) to your application and use the [psql](https://docs.postgresql.fr/10/app-psql.html) command.
+```bash
+ssh -t ssh@sshgateway-clevercloud-customers.services.clever-cloud.com <app_id>
+```
+```bash
+psql
+```
+
+#### How can I see the configured nodes?
+You can use the [`SHOW POOL_NODES;`](https://www.pgpool.net/docs/latest/en/html/sql-show-pool-nodes.html) command.
+```sql
+bjzfhkl5qc**********=> SHOW POOL_NODES;
+ node_id |                         hostname                          | port | status | lb_weight |  role   | select_cnt | load_balance_node | replication_delay | replication_state | replication_sync_state | last_status_change
+---------+-----------------------------------------------------------+------+--------+-----------+---------+------------+-------------------+-------------------+-------------------+------------------------+---------------------
+ 0       | bjzfhkl5qc**********-postgresql.services.clever-cloud.com | 57** | up     | 0.333333  | primary | 0          | false             | 0                 |                   |                        | 2021-06-02 19:54:53
+ 1       | b3diajq7iy**********-postgresql.services.clever-cloud.com | 57** | up     | 0.333333  | standby | 2          | false             | 0                 |                   |                        | 2021-06-02 19:54:53
+ 2       | bmtwtemn40**********-postgresql.services.clever-cloud.com | 57** | up     | 0.333333  | standby | 1          | true              | 0                 |                   |                        | 2021-06-02 19:54:53
+```
+
+#### How can I see the Pgpool-II processes waiting for connections and processing a connection?
+You can use the [`SHOW POOL_PROCESSES;`](https://www.pgpool.net/docs/latest/en/html/sql-show-pool-processes.html) command.
+```sql
+bjzfhkl5qc**********=> SHOW POOL_PROCESSES;
+ pool_pid |     start_time      |       database       |       username       |     create_time     | pool_counter
+----------+---------------------+----------------------+----------------------+---------------------+--------------
+ 21554    | 2021-06-03 11:28:36 | bjzfhkl5qc********** | u5dh6v2ymn********** | 2021-06-03 11:28:48 | 1
+ 11409    | 2021-06-02 19:54:53 |                      |                      |                     |
+ 11410    | 2021-06-02 19:54:53 |                      |                      |                     |
+ 21566    | 2021-06-03 11:29:20 |                      |                      |                     |
+```
+
+#### How can I see the list of pools managed by Pgpool-II?
+You can use the [`SHOW POOL_POOLS;`](https://www.pgpool.net/docs/latest/en/html/sql-show-pool-pools.html) command.
+```sql
+bjzfhkl5qckns3f2zfse=> SHOW POOL_POOLS;
```

Comment:
You did not hide the name here. I wouldn't have bothered doing it in the first place, it doesn't hurt to let it. But consistency :p